rally-domain/src/main/groovy/yakworks/rally/mail/model/MailMessage.groovy

```diff
@@ -85,10 +85,10 @@ class MailMessage implements UuidRepoEntity<MailMessage>, AuditCreatedTrait, Ser
     static enum MsgState {
         Queued, //Queued in the system, ready to be sent to mail processor
         Sent, //sent from here vai mailgun or smtp
-        Error, //an error occured and message failed
         Delivered, // if using something like mailgun and we interface, Mailgun sent the email and it was accepted by the recipient email server.
         Opened, // When using something like mailgun with Open tracking enabled
-        Complained //The email recipient clicked on the spam complaint button within their email client.
+        Complained, //The email recipient clicked on the spam complaint button within their email client.
+        Failed //an error occured and message failed, (could be mailgun permanent/temporary failure)
```

Comment:
@snimavat , this is the same as what you had before, you just renamed Error to Failed and now we dont have one for the temp failure. 
we need 2. Error and Failed. Error is when its a full Error. Failed is what you originally called TempFail.